temperatureHumidity.py

```diff
@@ -137,16 +135,16 @@ def logging():
     print(folderName)
     try:
         os.makedirs(folderName, mode=0o777)
+        print('Folder created')
     except OSError, e:
-	if e.errno != 17:
-        	print('Error creating folder: ' + str(e) ) # errno 17= File existed 
-	else: 
-		pass 
-    print('Folder created')
```

Comment:
Only print the message if makedirs() succeded. Otherwise we didn't create the folder.